model/subscriber.go

```diff
@@ -80,14 +81,14 @@ func (s *Subscriber) GetCache() datastore.Cache {
 func CreateSubscriber(ctx context.Context, store datastore.Store, s *Subscriber) error {
 	// If the subscriber has an ID, use that.
 	if s.ID != 0 {
-		key := store.IDKey(typeSubscriber, s.ID)
+		key := store.NameKey(typeSubscriber, fmt.Sprintf("%d.%s", s.ID, s.Email))
```

Comment:
It doesn't really make sense to me to have a randomly generated int plus an email for the key. Should we just use the email? That's going to be unique to the subscriber right? @scruzin might have some ideas.

Reply:
@scruzin mentioned previously that he liked the idea of using the ID for any user facing links, for example a public profile page might look like ```ausocean.tv/user/1029384756```
From a technical perspective however, I don't think there is any requirement to leave it in.